### PR TITLE
added a new itemadapter for 1.19

### PR DESCRIPTION
**Pull request by J3fftw1**
Updated dough to 1.19
Couldn't test building dough since mypet repo is down again.

Changed Worldedit repo to the enginehub maven repo
added a new ItemAdapter since they changed obf again

### CODE REVIEW

**Comment by J3fftw1** [comment]
Seems to be ready now. Thanks for the fix @TheBusyBiscuit

File: dough-items/src/main/java/io/github/bakedlibs/dough/items/nms/ItemNameAdapter19.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+import io.github.bakedlibs.dough.reflection.ReflectionUtils;
+import io.github.bakedlibs.dough.versions.UnknownServerVersionException;
+import org.bukkit.inventory.ItemStack;
+
+import javax.annotation.ParametersAreNonnullByDefault;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+
+public class ItemNameAdapter19 implements ItemNameAdapter {
```
Comment (TheBusyBiscuit) [inline dough-items/src/main/java/io/github/bakedlibs/dough/items/nms/ItemNameAdapter19.java:11]: This should not be public.
```suggestion
class ItemNameAdapter19 implements ItemNameAdapter {
```

**Review by TheBusyBiscuit — CHANGES_REQUESTED** [review]

**Review by WalshyDev — APPROVED** [review]

**Review by md5sha256 — APPROVED** [review]
LGTM

**Review by TheBusyBiscuit — APPROVED** [review]
Looks good to me.
Just before I merge this: Does `dough-skins` not need an update for its NMS code?

**Comment by J3fftw1** [comment]
> Looks good to me. Just before I merge this: Does `dough-skins` not need an update for its NMS code?

Nope it doesnt. We specifically checked for this.

**Pull request merged**